variables.tf

```diff
@@ -72,7 +72,7 @@ variable "performance_mode" {
 }
 
 variable "provisioned_throughput_in_mibps" {
-  default     = ""
+  default     = 1.0
```

Comment:
I wonder if we should try `0.0` instead. It's odd that TF (or AWS) is complaining when `throughput_mode` is defaulted to `bursting`.